dts/bindings/gpio/silabs,siwx917-gpio.yaml

```diff
@@ -0,0 +1,17 @@
+description: SiLabs SiWx917 GPIO node
+
+compatible: "silabs,siwx917-gpio"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  silabs,ulp:
+    type: boolean
+    description: |
+      If this property is set, the GPIO uses ULP pads.
```

Comment:
I believe it would help the user if we expanded the "ULP" acronym here (ditto for UULP)

Reply:
Done.